src/components/Admin/WikiEditorInsight/WikiEditorsInsight.tsx

```diff
@@ -53,6 +36,17 @@ export const WikiEditorsInsightTable = () => {
     id: string
     active: boolean
   }>({ id: '', active: false })
+  const [activatePrevious, setActivatePrevious] = useState<boolean>(false)
+  const [allowNext, setAllowNext] = useState<boolean>(true)
```

Comment:
useState can infer from initial value ! so don't need to pass the type in the generic

```suggestion
  const [activatePrevious, setActivatePrevious] = useState(false)
  const [allowNext, setAllowNext] = useState(true)
```